crates/rattler_solve/src/lib.rs

```diff
@@ -227,16 +227,16 @@ mod test_libsolv {
         assert!(matches!(operations[0].kind, PackageOperationKind::Install));
         let info = &operations[0].package;
 
-        assert_eq!("foo-3.0.2.tar.bz2", info.file_name);
+        assert_eq!("foo-3.0.2.conda", info.file_name);
         assert_eq!(
-            "https://conda.anaconda.org/conda-forge/linux-64/foo-3.0.2.tar.bz2",
+            "https://conda.anaconda.org/conda-forge/linux-64/foo-3.0.2.conda",
```

Comment:
I didn't pay attention to this before, but this is actually an "invalid" package name. It should be `<NAME>-<VERSION>-<BUILDSTRING>.<EXTENSION>`. The `package_record.build` also looks weierd (with the `_conda`).

Reply:
Yeah, that is a consequence of my unfamiliarity with the conda ecosystem. I'll update it to use more realistic values